preludezdev/ArchitectureStudy/app/src/main/java/com/example/architecturestudy/ui/main/MainActivity.kt

```diff
@@ -20,11 +20,14 @@ class MainActivity : BaseActivity<ActivityMainBinding>(R.layout.activity_main) {
     }
 
     private fun initViewPager() {
-        //뷰페이저 어댑터 연결
-        binding.viewPager.adapter = vpAdapter
-
-        //탭 레이아웃에 뷰페이저 연결
-        binding.tabLayout.setupWithViewPager(binding.viewPager)
+        with(binding) {
+            //뷰페이저 어댑터 연결
+            viewPager.adapter = vpAdapter
+            viewPager.offscreenPageLimit = vpAdapter.count
+
+            //탭 레이아웃에 뷰페이저 연결
+            tabLayout.setupWithViewPager(binding.viewPager)
```

Comment:
binding 변수는 생략해도 될 것 같습니다.
```suggestion
            tabLayout.setupWithViewPager(viewPager)
```

Reply:
안녕하세요 현석님, 피드백 해주셔서 정말 감사합니다 
저도 잘 부탁드립니다 :)

해당부분 수정했습니다

commit : 33f5232

Reply:
안녕하세요 현석님, 피드백해주셔서 감사합니다.
저도 잘 부탁드릴게요 :D

라고 분명히 저번에 답글 달때 이렇게 올렸는데 답글이 안달려있네요 ;( 
여튼 다시 한번 피드백 감사드립니다!!

해당 부분 반영했습니다.
commit : 33f5232